source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -47,6 +47,9 @@ using Http::ResponseTrailerMap;
 constexpr absl::string_view ErrorPrefix = "ext_proc_error";
 constexpr int DefaultImmediateStatus = 200;
 constexpr absl::string_view FilterName = "envoy.filters.http.ext_proc";
+constexpr absl::string_view RemoteCloseTimeout =
+    "envoy.filters.http.ext_proc.remote_close_timeout_milliseconds";
```

Comment:
can this be added to a runtime section for ext proc? e.g. https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/ext_proc_filter similar to https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/fault_filter#runtime

Reply:
Done